docs/integrations/snyk.md

```diff
@@ -1,38 +1,71 @@
+![Setup with existing group](../../assets/snyk_setup_existing_group.png)
 
-Once you have authenticated, you will need to accept the permissions required to run the Phylum application. These permissions will grant Phylum access to your projects and dependencies list.
+To import projects from Snyk Phylum needs a Snyk API key. This key is not only
+used for authentication, but also decides which Snyk projects should be
+imported. All projects accessible to the API key will be imported, so creating a
+Snyk service account with the desired access permissions is recommended.
 
-![Accept permissions and scopes](../../assets/snyk_ask_scopes.png)
+Instructions on setting up a service account can be found in Snyk's
+documentation:
 
-## Listing Projects
-After installing the Phylum integration for Snyk, you can now list your Snyk projects by clicking the "List Projects" button.
+https://docs.snyk.io/enterprise-configuration/service-accounts#how-to-set-up-a-group-or-organization-service-account
 
-![Listing Snyk Projects](../../assets/snyk_list_projects.png)
+![Token entry](../../assets/snyk_setup_token.png)
 
-Once you've installed the application, you will need to link your Snyk account to Phylum by clicking on the `Enable` button in the interface.
+## Updates
 
-![Enable with Phylum](../../assets/snyk_enable.png)
+An import of all Snyk projects is automatically performed after initial setup.
+Once setup, existing connections are re-imported once a day.
 
-If you aren't already authenticated with Phylum, you will be prompted to do so now.
+Imports run asynchronously, so it might take a while to finish and projects will
+not necessarily all show up at the same time. Phylum will show a message if an
+import is not completed yet.
 
-![Phylum Authenticate](../../assets/snyk_phylum_auth.png)
+![Import in progress](../../assets/snyk_import_in_progress.png)
 
-## Analyzing Results
-Phylum analyzes software packages as they are published into the open source ecosystem. In addition to providing information around software vulnerabilities and licenses, we can also provide information around real-time threats stemming from your software supply chain.
+Manual imports can be started at any time if the daily update is not sufficient.
+This can be done by pressing the `Refresh Imports` button.
 
-By clicking on a project you can view additional supply chain attack information related to your project dependencies.
+![Manual import](../../assets/snyk_manual_import.png)
 
-![Anaylze Supply Chain Attacks](../../assets/snyk_supply_chain_attacks.png)
+## Snyk Token Updates
+
+Phylum will notify you if the associated token is revoked or invalidated.
+
+![Invalid token](../../assets/snyk_invalid_token.png)
+
+In this case, the token can be updated by just pressing the red `Update Token`
+button.
+
+![Invalid token update](../../assets/snyk_invalid_token_update.png)
+
+If the token is still valid, a manual change can be performed by selecting
```

Comment:
Is this supposed to say invalid?

Reply:
Yeah, it's documenting the update of an invalid token? I'm confused about your question.

Reply:
Sorry, it says: "If the token is still valid"... isn't this supposed to say: "If the token is still invalid"?

Reply:
I think the "still" is what threw me off. It feels like a continuation from the previous instructions, instead of being a new instruction.